### ISSUE TITLE
I guess it's not "S" part but "T" part are not in this distribution.

### ISSUE THREAD

**Issue by JasperWYk** [rottenivy/ST-Matching#4]

**Comment by happyBayes** [comment]
I agree with you ,since the temporal function that judges road restriction of edges is unimplemented in this distribution

**Comment by xulinaxulixiang** [comment]
Will the above code work successfully? My three road network files are all in csv format, and then each column is set according to the readme. The trajectory file is also in csv format. The three columns included are traj_id, lan, lon, but the operation shows an error, where can I be wrong?

**Comment by laozheng1** [comment]
I have the same question. Is it running successfully?

**Comment by banana7778** [comment]
> 我有同样的问题。它是否成功运行？

hi i wonder didU made this successful?
